framework/PostProcessors/ParetoFrontierPostProcessor.py

```diff
@@ -65,11 +67,21 @@ def _handleInput(self, paramInput):
       @ In, paramInput, ParameterInput, the already-parsed input.
       @ Out, None
     """
+    self.valueLimit = None
+    self.costLimit  = None
+
     costID  = paramInput.findFirst('costID')
     self.costID  = costID.value
     valueID = paramInput.findFirst('valueID')
     self.valueID = valueID.value
 
+    costLimit  = paramInput.findFirst('costLimit')
+    if costLimit is not None:
+      self.costLimit  = float(costLimit.value)
+    valueLimit = paramInput.findFirst('valueLimit')
+    if valueLimit is not None:
+      self.valueLimit = float(valueLimit.value)
```

Comment:
If you use ```contentType=InputTypes.FloatType```, you could avoid the converting.

Reply:
Gotcha, edited